help_to_heat/frontdoor/views.py

```diff
@@ -322,6 +323,12 @@ def get(self, request, session_id, page_name, extra_data=None, errors=None, is_c
             else:
                 return redirect("/sorry")
 
+        old_data = data.copy()
+        data = self.save_get_data(data, session_id, page_name)
+        # only save an answer on get if new answers were given
+        if data != old_data:
+            save_answer(session_id, page_name, data)
```

Comment:
What's this bit doing?

Reply:
there are a couple pages that need to save their answers on get

however, this only happens on 3 pages, so saving answers always would save a lot of extra answers to the database. here it checks if anything happened to the data object before saving

Reply:
Which are these pages? Feels weird to be saving something on GET (should be for fetching not for updates)

Reply:
Ah right, analytics things, ok.